IdentityCore/src/cache/mac/MSIDMacKeychainTokenCache.m

```diff
@@ -474,30 +536,27 @@ - (MSIDCredentialCacheItem *)tokenWithKey:(__unused MSIDCacheKey *)key
                                               context:(__unused id<MSIDRequestContext>)context
                                                 error:(__unused NSError **)error
 {
-    NSArray *tokenItems = [self itemsWithKey:key context:context error:error];
-
-    if (!tokenItems)
+    NSArray<MSIDCredentialCacheItem *> *tokenList = [[NSArray alloc] init];
+    
+    if (key.isShared)
     {
-        return nil;
+        MSIDMacSharedCredentialCacheItem *sharedCredential = [self sharedCredentialWithKey:key context:context error:error];
+        
+        if (sharedCredential)
+        {
+            tokenList = [sharedCredential sharedCredentialsWithKey:(MSIDDefaultCredentialCacheKey *)key];
```

Comment:
verify you get default credential cache key

Reply:
Done